### ISSUE TITLE
UI not showing up

### ISSUE THREAD

**Issue by danixele** [taniwha/Extraplanetary-Launchpads#151]
Basically, when I press "show UI" from the workshop or when I press the button from the toolbar, the menu comes up but it doesn't load the elements inside, like the pad selector or the GUI's buttons. I'm having this problem since I landed on Mun, as before taking off from Kerbin the mod actually worked! Am I missing something?

I apologize for my bad english,
Hope the attachment will help to understand the problem.

![20190514141113_1](https://user-images.githubusercontent.com/50623634/57697291-2c5a7680-7653-11e9-9972-654fe276284f.jpg)

**Comment by 4pony** [comment]
I'm finding the same thing.  I did some digging and I discovered that I can make the Survey Station work, but only when it's a fresh load and the Survey Station ship doesn't have any of the Kerbal Attachment System resource transfer gadgets as part of the build.  I can go EVA, attach an RTS-1, use it in docked mode to transfer resources, but I must then detach the RTS-1 after use.  If I leave it in place, on the next reload, the Survey Station will be non-functional.  Its range drops to 20m, it cannot see any of the stakes, and if I attempt to load a craft via the build UI, the UI goes blank (apart from the highlight button).

However, if I remove the KAS parts via EVA, save and reload, the Survey Station becomes functional again.

I've tried the remove KAS, save and subsequent reload trick with the big launchpad, but I've yet to see it work at all.

**Comment by Teslamax** [comment]
I have this issue as well.

If I remove the offending `KS-MP Disposable Pad` and replace it using KAS it has cleared up.

```
[EXC 01:51:53.950] NullReferenceException: Object reference not set to an instance of an object
    ExtraplanetaryLaunchpads.ELBuildWindow.RequiredResources ()
    ExtraplanetaryLaunchpads.ELBuildWindow.WindowGUI (Int32 windowID)
    UnityEngine.GUILayout+LayoutedWindow.DoWindow (Int32 windowID)
    UnityEngine.GUI.CallWindowDelegate (UnityEngine.WindowFunction func, Int32 id, Int32 instanceID, UnityEngine.GUISkin _skin, Int32 forceRect, Single width, Single height, UnityEngine.GUIStyle style)
[ERR 01:51:53.950] GUI Error: You are pushing more GUIClips than you are popping. Make sure they are balanced)

[EXC 01:51:53.952] NullReferenceException: Object reference not set to an instance of an object
    ExtraplanetaryLaunchpads.ELBuildWindow.RequiredResources ()
    ExtraplanetaryLaunchpads.ELBuildWindow.WindowGUI (Int32 windowID)
    UnityEngine.GUILayout+LayoutedWindow.DoWindow (Int32 windowID)
    UnityEngine.GUI.CallWindowDelegate (UnityEngine.WindowFunction func, Int32 id, Int32 instanceID, UnityEngine.GUISkin _skin, Int32 forceRect, Single width, Single height, UnityEngine.GUIStyle style)
[ERR 01:51:53.952] GUI Error: You are pushing more GUIClips than you are popping. Make sure they are balanced)
```

**Comment by stargazer424** [comment]
I found a possible reason for this issue. EL and KAS are having issues with each other. If you detach and re-attach all KAS items (Joint Sockets, Resource Transfer stations) it works again...

https://www.youtube.com/watch?v=FsBDFpmE2g8

**Comment by toastom** [comment]
Hi there. I'm having the same trouble, but also with my survey stakes exploding when trying to load a new craft. What version is the latest for EL? The latest I could find is 6.5.1 and I'm wondering if I'm having some sort of conflict with KAS from it.

**Comment by taniwha** [comment]
Fixed by commit 7c8e75a
